### PR TITLE
Make endpoint OrderModel for admin user

### PR DESCRIPTION
**Pull request by VladPrime11**

### CODE REVIEW

File: core/urls.py [hunk 1]
```diff
@@ -5,5 +5,7 @@
 
 urlpatterns = [
     path('api/admin/', admin.site.urls),
-    path('api/shop/', include('apps.shop.urls'))
+    path('api/shop/', include('apps.shop.urls')),
+    path('apps/shop/', include('apps.shop.urls'))
```
Comment (HomaDev) [inline core/urls.py]: What is the purpose of apps/shop/ we have api/shop/

File: apps/shop/views.py [hunk 2]
```diff
@@ -52,3 +52,9 @@ def list(self, request):
             raise APIException
         serializer = ShopContactsSerializer(queryset)
         return Response(serializer.data)
+
+
+class OrderViewSet(viewsets.ModelViewSet):
+    queryset = OrderModel.objects.all()
+    serializer_class = OrderSerializer
+    permission_classes = [permissions.IsAdminUser]
```
Comment (HomaDev) [inline apps/shop/views.py]: Let's stick to one Views design that was used before, I mean ViewSet. 
It gives more flexibility over the project, If we want to use ModelViewSet will need to refactor all existing ViewSet Views with ModelViewSet

File: core/urls.py (hunk 1 above)
Comment (HomaDev) [reply core/urls.py]: @AntonPrime11 
Still cant see?

File: apps/shop/views.py (hunk 2 above)
Comment (VladPrime11) [reply apps/shop/views.py]: > Let's stick to one Views design that was used before, I mean ViewSet. It gives more flexibility over the project, If we want to use ModelViewSet will need to refactor all existing ViewSet Views with ModelViewSet

Meaning, make access not just for administrators?

Comment (HomaDev) [reply apps/shop/views.py]: Mean change OrderViewSet to regular ViewSet to follow existing design
read/patch/delete/update access for administrators and post for everybody.

**Comment by HomaDev** [comment]
I have to close it, because out design have significantly changed

**Pull request closed without merging**